pom.xml

```diff
@@ -141,6 +141,14 @@
 			<artifactId>jackson-databind</artifactId>
 		</dependency>
 
+		<!-- Utilities -->
+		<dependency>
+			<groupId>org.apache.commons</groupId>
+			<artifactId>commons-collections4</artifactId>
+			<version>4.4</version>
+		</dependency>
+
+
```

Comment:
Será que precisamos mesmo desta classe (veja os comentários na classe BussinessValidation)

Reply:
Podemos remover se for utilizar o Assert do Spring. Vou avaliar